src/main/kotlin/com/cognifide/gradle/aem/pkg/ComposeTask.kt

```diff
@@ -176,8 +180,12 @@ open class ComposeTask : Zip(), AemTask {
     fun includeBundles(project: Project) {
         dependProject(project)
 
+        val extraProperties = project.extensions.extraProperties
+        val bundlePath: String = if (extraProperties == null || !extraProperties.has("bundlePath")) config.bundlePath
+        else extraProperties.get("bundlePath") as String
+
         bundleCollectors += {
-            JarCollector(project).all.toList()
+            Pair(bundlePath, JarCollector(project).all.toList())
```

Comment:
but can have, because it has applied AEM plugin which works with both bundle and content projects;
in this file https://github.com/Cognifide/gradle-aem-example/blob/master/bundle/build.gradle
you could have aemCompose { config { bundlePath = 'xxx' } } section